packages/api/src/configuration/index.ts

```diff
@@ -45,24 +45,24 @@ export enum MAX {
   PROPERTY = 24,
   EXPIRESIN = 4,
   ENV = 12,
-  REF = 100,
+  REF = 500,
   CLUSTER = 12
 }
 
 export enum MESSAGE {
-  INVALID_LENGTH = 'Length is Invalid.',
-  INVALID_PROPERTY_TITLE = 'Invalid Property Title.',
-  INVALID_PROPERTY_IDENTIFIER = 'Invalid Property Identifier.',
+  INVALID_LENGTH = 'Length is Invalid',
+  INVALID_PROPERTY_TITLE = 'Invalid Property Title [Example : Test Property].',
+  INVALID_PROPERTY_IDENTIFIER = 'Invalid Property Identifier [Example : test-property].',
   INVALID_CLUSTER = 'Invalid Cluster [Must be Prod/Preprod].',
   INVALID_EPHEMREAL = 'Invalid Ephemeral [Must be true/false].',
   INVALID_PATH = 'Invalid Path [Example : /home, /test-app].',
   INVALID_URL = 'Invalid URL [Example : spaship.redhat.com].',
   INVALID_EXPIRESIN = 'Invalid ExpiresIn [Example : 1d, 5d, 100d].',
-  INVALID_ACTION_ID = 'Invalid Action Id [Please provide the Pull Request ID, Ex: 101].',
-  INVALID_APPLICATION = 'Invalid Application name.',
-  INVALID_REF = 'Invalid Reference.',
-  INVALID_ENV = 'Invalid Environment',
-  INVALID_LABEL = 'Invalid Label.'
+  INVALID_ACTION_ID = 'Invalid Action Id [Please provide the Pull Request ID, Example: 101].',
+  INVALID_APPLICATION = 'Invalid Application name [Example : home, test, redhat doc].',
+  INVALID_REF = 'Invalid Reference [Example : 1.2, v1@1.2.3].',
+  INVALID_ENV = 'Invalid Environment [Example : prod, stage, dev]',
+  INVALID_LABEL = 'Invalid Label [Example : all-access, prod-key].'
```

Comment:
I think we can come up with better error response than this

Reply:
Any better format ?